src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx

```diff
@@ -111,15 +111,14 @@ however it is no longer tested or officially supported by the New Relic PHP Agen
       </td>
 
       <td>
-        * Amazon Linux 2 (including AWS Graviton 2)
-        * CentOS 8
-
           <Callout variant="important">
-            ARM64 is only supported with:
+            ARM64 is supported on Amazon Linux 2 (including AWS Graviton 2) and Centos 8 with the following agents ONLY when compiled from source:
+
+            * PHP 8.0 and with New Relic PHP Agent 9.18.1 - 10.9.0.
+            * PHP 8.1 and with New Relic PHP Agent 9.19.0 - 10.9.0.
+            * PHP 8.2 and with New Relic PHP Agent 10.7.0 - 10.9.0.
 
-            * PHP 8.0 and with New Relic PHP Agent 9.18.1 or later.
-            * PHP 8.1 and with New Relic PHP Agent 9.19.0 or later.
-            * PHP 8.2 and with New Relic PHP Agent 10.7.0 or later.
```

Comment:
I would keep this page as it was without any changes. Maybe add a separate call out section that New Relic PHP Agent 10.10.0 or later provides binaries for ARM64. Detailed information about ARM64 should be on the php-agent-installation-arm64 page.